### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by s3aryal**

### CODE REVIEW

**Comment by christikaes** [comment]
Awesome! 🎉 

I don't see any new cypress tests for this work here https://dashboard.cypress.io/projects/34dwxm/runs/6/specs. I'd like to make sure that we are testing the code as we are developing it to reduce possible future regressions. Let us know if you need any help with writing tests!

File: server/server.js [hunk 1]
```diff
@@ -2,35 +2,62 @@ const express = require('express')
...
+	res.send(org)
+})
 
-	organizations[id_val] = {'Name':req.body.name};
-	res.sendStatus(200);
+app.delete('/organization_id', function (req, res) {
+    var id = req.query.id;
+    let org = organizations[id]
+    delete organizations[id]
+    res.sendStatus(200)
```
Comment (binamkayastha) [inline server/server.js]: Issue: https://github.com/nepalcodes/unitednepali/issues/11
Here's what you'd need to change to make it compliant to the issue:
```suggestion
    if (org) {
      res.status(200)
      res.send({
        "success": 1,
        "timestamp": new Date().getTime()
      })
    } else {
      res.status(500)
      res.send({
        "success": 0,
        "timestamp": new Date().getTime()
        "error": "Organization with id " + id + " does not exist!"
      })
   }
```
I just wrote this in a comment and haven't tested this out, so there might be some bugs here :P

File: server/server.js [hunk 2]
```diff
@@ -2,35 +2,62 @@ const express = require('express')
...
+})
 
+app.post('/organization_id', function(req, res) {
+	var id = req.query.id;
+	organizations[id] = req.query;
+	res.sendStatus(200);
```
Comment (binamkayastha) [inline server/server.js]: Issue: https://github.com/nepalcodes/unitednepali/issues/9

File: server/server.js [hunk 3]
```diff
@@ -2,35 +2,62 @@ const express = require('express')
...
+	var org = organizations[id];
+
+	for(var key in req.query){
+		if (key in org){
+			org[key] = req.query[key];
+		}
+	}
+	organizations[id] = org;
+
+	res.sendStatus(200);
```
Comment (binamkayastha) [inline server/server.js]: Issue: https://github.com/nepalcodes/unitednepali/issues/10

File: server/server.js [hunk 4]
```diff
@@ -2,35 +2,62 @@ const express = require('express')
...
 
 app.get('/organizations', (req, res) => res.send(organizations))
 
-app.post('/organizations', function(req, res){
-	let id_val = currId;
-	currId = currId + 1;
+app.get('/organizations_id', function (req, res) {
+	var id = req.query.id;
+	let org = organizations[id]
+	res.send(org)
```
Comment (binamkayastha) [inline server/server.js]: Issue: https://github.com/nepalcodes/unitednepali/issues/8

**Comment by christikaes** [comment]
Great job adding tests, let's merge this in now and we can continue iterating in the future

**Pull request merged**